src/Tooltip.js

```diff
@@ -163,27 +170,101 @@ export default class Tooltip extends BaseClass {
 
     divElement("footer");
 
+    divElement("arrow");
+
     enter.call(boxStyles);
 
     const t = transition().duration(this._duration);
 
     update
-      .attr("id", (d, i) => `d3plus-tooltip-${this._id(d, i)}`)
+      .attr("id", (d, i) => `d3plus-tooltip-${d ? this._id(d, i) : ""}`)
       .transition(t)
-        .style(`${this._prefix}transform`, "scale(1)")
+        .style("opacity", 1)
         .call(boxStyles);
 
     tooltips.exit()
       .transition(t)
-        .style(`${this._prefix}transform`, "scale(0)")
-        .remove();
+        .style("opacity", 0)
+      .remove();
+
+    for (let i = 0; i < this.data.length; i++) {
+      if (that._data[i]) {
+        const tooltip = document.getElementById(`d3plus-tooltip-${that._id(that._data[i], i)}`);
+        const arrow = document.getElementById(`d3plus-tooltip-arrow-${that._id(that._data[i], i)}`);
+        const arrowHeight = arrow.getBoundingClientRect().height;
+        arrow.style.bottom = `-${arrowHeight / 2 + 1}px`;
+
+        const arrowOffset = arrowHeight / 4;
+
+        const referenceObject = that._position() instanceof Element ? that._position() : {
```

Comment:
This conditional block was erroring out for me when trying to supply an accessor function to an array of 2 values. I've never seen the `Element` comparitor, maybe we should just use an Array comparitor?

```jsx
const position = that._position(that._data[i]);
const referenceObject = position instanceof Array ? {
  clientWidth: 0,
  clientHeight: 0,
  getBoundingClientRect: () => ({
    top: that._position(that._data[i])[1] - arrowOffset,
    right: that._position(that._data[i])[0] - arrowOffset,
    bottom: that._position(that._data[i])[1] - arrowOffset,
    left: that._position(that._data[i])[0] - arrowOffset,
    width: 0,
    height: 0
  })
} : position;
```